### PR TITLE
fix kmeshctl auth command description

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: ctl/authz/authz.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func NewCmd() *cobra.Command {
  kmeshctl authz <kmesh-daemon-pod> enable
  
  # Disable Kmesh's authz:
- kmeshctl authz <kmesh-daemon-pod> authz`,
+ kmeshctl authz <kmesh-daemon-pod> disable`,
```
Comment (YaoZengzeng) [inline ctl/authz/authz.go:46]: Seems need to use `hack/gen-kmeshctl-doc.sh` to update doc of kmeshctl

File: ctl/authz/authz.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ func NewCmd() *cobra.Command {
  kmeshctl authz <kmesh-daemon-pod> enable
```
Comment (hzxuzhonghu) [inline ctl/authz/authz.go:43]: can you add more comments,  `Enable or disable Kmesh's authz offloading`

File: ctl/authz/authz.go [hunk 3]
```diff
@@ -39,11 +39,11 @@ func NewCmd() *cobra.Command {
...
 		Use:   "authz",
 		Short: "Enable or disable Kmesh's authz",
-		Example: `# Enable Kmesh's authz:
- kmeshctl authz <kmesh-daemon-pod> enable
+		Example: `# Enable/Disable Kmesh's authz in the specified kmesh daemon:
+ kmeshctl authz <kmesh-daemon-pod> enable/disable
  
- # Disable Kmesh's authz:
- kmeshctl authz <kmesh-daemon-pod> authz`,
+ # If you want to enable or disable authz offloading of all Kmeshs in the cluster
```
Comment (hzxuzhonghu) [inline ctl/authz/authz.go:45]: Actually you need to add it to both `Short` and Exmaple

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**